### PR TITLE
16.0 mig stock picking customer ref

### PR DESCRIPTION
**Pull request by cvinh**
Nothing to do, test migrated

### CODE REVIEW

**Comment by cvinh** [comment]
@alfredoavanzosc please review that PR

**Review by oihane — APPROVED** [review]

File: stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Copyright 2015 AvanzOSC - Alfredo de la Fuente
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestStockPickingCustomerRef(common.SavepointCase):
```
Comment (rousseldenis) [inline stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py]: Could you change this to TransactionCase ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor change

File: stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py (hunk 1 above)
Comment (cvinh) [reply stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py]: Done

**Review by hbrunn — APPROVED** [review]
@rousseldenis please update your review

**Review by gaikaz — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**